PMT type positions

mnieslony opened the pull request:
The locations for the different PMT types were adjusted to match the real ANNIE detector configuration more closely. In this regard, the bottom PMTs were rotated by 45 degrees and the PMT type mapping for the barrel PMTs was adjusted.

mnieslony committed:
added new PMT type positionings

marc1uk commented:
Sorry for having left this in limbo forever, and thanks for fixing this. I did check it over and noticed that although the bottom cap PMTs have been rotated, the inner structure hasn't which results in a mismatch in orientation. 
![trash](https://user-images.githubusercontent.com/15327579/99527122-31a92780-2994-11eb-949c-aabf18596baa.png)
Given where the PMTs are placed relative to the inner structure, there actually isn't any overlap, which wouldn't have resulted in any warnings from the Constructor. That's probably why it went unnoticed, and it probably wouldn't have much impact (just affecting reflections) but a little further down and you'd get overlaps which would cause problems tracking (although in that case WCSim ought to warn you about those overlaps). 
![trash](https://user-images.githubusercontent.com/15327579/99527508-ce6bc500-2994-11eb-9217-9342bd6950cd.png)

Changing src/ConstructANNIE.cc::222 to `rotm2->rotateZ(67.5*deg);` would align the inner structure properly.

marc1uk committed:
fix inner structure rotation

rotate gdml inner structure in line with rotation of bottom cap PMTs introduced in the last commit.

marc1uk commented:
fixed and merged. :+1:
